app/controllers/ViewAttachmentController.scala

```diff
@@ -43,6 +45,16 @@ class ViewAttachmentController @Inject() (
     with I18nSupport
     with WithUnsafeDefaultFormBinding {
 
+  /*
+   * This method is used to encode the filename in the Content-Disposition header
+   * url encode the filename and browser will decoded based on the charset you specify
+   * if no filename is provided, generate a random one
+   */
+  private def getEncodedFilenameHeader(fileSubmitted: FileMetadata): String =
+    fileSubmitted.fileName
+      .map(fileName => s"filename*=UTF-8''${URLEncoder.encode(fileName, "UTF-8")}")
```

Comment:
I did see some examples where they would keep the `filename=` as well as `filename*=` as a fallback for older browser support. Should we expect to maintain older support like that ?

Example: [Mozilla Content-Disposition - Compatibility notes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition#compatibility_notes)

Reply:
this is internal app when I was part of ATAR team the gov staff were moving slowing to edge and edge is chrome basically so I dont expect any problems at all here, and also if you look in notes there on mozila you see firefix handles this since v5 so its been over a decade

Reply:
Makes sense thanks